eth/stagedsync/stage_senders.go

```diff
@@ -345,6 +347,8 @@ func UnwindSendersStage(s *UnwindState, tx kv.RwTx, cfg SendersCfg, ctx context.
 }
 
 func PruneSendersStage(s *PruneState, tx kv.RwTx, cfg SendersCfg, ctx context.Context) (err error) {
+	to := cfg.prune.TxIndex.PruneTo(s.ForwardProgress)
```

Comment:
In Silkwarm senders also belong to TxIndex prune flag?

Reply:
No, silkworm as most things is very much in a pre-alpha stage so there is no optional pruning there, you either prune all or you prune nothing